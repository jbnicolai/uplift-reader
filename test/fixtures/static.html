<!DOCTYPE html>
<html class="no-js" lang="">
<head>
  <title></title>

  <link rel="shortcut icon" href="/static/img/favicon.png">

  <!-- build:css /static/css/app.min.css -->
  <link rel="stylesheet" href="/static/css/app.css">
  <!-- endbuild -->

</head>
<body>

  <!-- build:js /static/js/app.min.js -->
  <script src="/static/js/app.js"></script>
  <!-- endbuild -->

  <script>console.log('random src="to mess with you"');</script>
</body>
</html>
