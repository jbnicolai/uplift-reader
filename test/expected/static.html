<!DOCTYPE html>
<html class="no-js" lang="">
<head>
  <title></title>

  <link rel="shortcut icon" href="/static/img/favicon.png">

  <link rel="stylesheet" href="/static/css/app.min.css"/>

</head>
<body>

  <script type="text/javascript" src="/static/js/app.min.js"></script>

  <script>console.log('random src="to mess with you"');</script>
</body>
</html>
